WORKSPACE

```diff
@@ -1,57 +1,80 @@
-workspace(name = "angular_cli")
+workspace(
+    name = "angular_cli",
+    managed_directories = {"@npm": ["node_modules"]},
+)
 
-load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
 
 http_archive(
     name = "build_bazel_rules_nodejs",
-    sha256 = "fb87ed5965cef93188af9a7287511639403f4b0da418961ce6defb9dcf658f51",
-    urls = ["https://github.com/bazelbuild/rules_nodejs/releases/download/0.27.7/rules_nodejs-0.27.7.tar.gz"],
+    # patch_args = ["-p1"],
+    # Patch https://github.com/bazelbuild/rules_nodejs/pull/903
+    # patches = ["//tools:rollup_bundle_commonjs_ignoreGlobal.patch"],
+    sha256 = "6625259f9f77ef90d795d20df1d0385d9b3ce63b6619325f702b6358abb4ab33",
+    urls = ["https://github.com/bazelbuild/rules_nodejs/releases/download/0.35.0/rules_nodejs-0.35.0.tar.gz"],
 )
 
 # We use protocol buffers for the Build Event Protocol
```

Comment:
Is this necessary still for BEP setup?  I know that we don't need to do this for `angular/angular` so I am not sure if we need to do this here.  Unless there is something additional I am not aware of.

Reply:
The CLI has some basic BEP streaming support and there is a utility that supports debugging it which uses the `proto_library` rule that in turn requires this.